content/about/agreements.md

```diff
@@ -7,138 +7,222 @@ $order: 5
+1. In the event it seems to me that you are not meeting our agreements, I will candidly and compassionate share my perspective with you, owning my part and acknowledging all stories and feelings as such.  Such conversations may lead to changes in both of our perspectives, behaviors, and roles. 
 
-#### Paper writing
+I expect to meet all of these commitments 90% of the time.  I hope that whenever I fail to meet any of these commitments in any way, that you share your perspective with me, so that I may learn and grow from your perspective, no matter what it is. 
 
-When it is time to start writing a paper, clone our [latex template](https://www.overleaf.com/read/mnmbyrtgphkp) overleaf repo, invite jovo to it and make him owner so that compilation goes faster and we can track changes. If a collaborator demands google docs for some reason, and we are not senior authors, then we will have to acquiese to their wishes. In general, when anybody gives feedback on the paper, make sure track changes is on. I recommend giving feedback via the following convention: 
 
-% TODO blah blah ---jovo
+## NeuroData Family
 
-This enables everyone to easily find all the comments, including if they are editing locally (or simply not in the overleaf web app). When feedback is provided, it is important to address every single comment *in the text itself*. Recall: the author of a manuscript is literally the least qualified person in the world to ascertain whether what that person has written is *clear to other people*, which is the question at hand. Therefore, if you are the author, and you think it was clear, *you are wrong*, in that it certainly was not clear to at least one person that cares. 
+The following commitments and agreements apply to all current members of the [NeuroData family](https://neurodata.io/about/team/), Jovo included
 
-#### Cloud computing
+### Commitments
 
-Everyone is welcome to use either AWS or Azure as desired. Currently, we have free Azure credits, so that is the preference. For AWS, if you want to use it, the prerequisite is writing a [AWS Cloud Credits for Research grant](https://aws.amazon.com/research-credits/). Request just under $10,000 for small projects, and request up to $100,000 for larger ones. For the Brainlit project, we have an NIH grant, and that is what we should be using entirely until it runs out of money. However, anyone who is using cloud computing should not run any code that runs overnight without verifying the code is correct. It must be run on small parameters/dataset, and a figure must be generated on that small run which needs to be approved by the PI.
+#### Genius
 
+> I commit to focusing the majority of my research [energy](http://a.co/aPrBEkq) on a single project in my  [zone of genius](https://www.amazon.com/dp/B07S7L6FV2?ref=KC_GS_GB_US), and to support and inspire others to fully express their creativity and live in their zone of genius.
```

Comment:
I'm also pro 1-2 projects

Reply:
I changed it because I am trying to be *less* controlling.  I guess I meant to write something more like:

> I commit to focusing the majority of my research [energy](http://a.co/aPrBEkq) on research projects in my  [zone of genius](https://www.amazon.com/dp/B07S7L6FV2?ref=KC_GS_GB_US).

But maybe you are right, better to say 'a few' rather than 'a single'?